pkg/kubelet/volume_host.go

```diff
@@ -69,8 +69,8 @@ func NewInitializedVolumePluginMgr(
 	const resyncPeriod = 0
 	if utilfeature.DefaultFeatureGate.Enabled(features.CSIDriverRegistry) {
 		// Don't initialize if kubeClient is nil
```

Comment:
Nit: the comment is not updated to heartbeatclient

Reply:
this comment is removed as what's being discussed in it is self-explanatory from code